src/components/scenes/newAccount/NewAccountTosScene.tsx

```diff
@@ -173,9 +174,9 @@ const getStyles = cacheStyles((theme: Theme) => ({
 export const NewAccountTosScene = connect<StateProps, DispatchProps, OwnProps>(
   (state: RootState) => ({
     createErrorMessage: state.create.createErrorMessage,
-    password: state.create.password || '',
+    password: state.create.password || null,
```

Comment:
Isn't the `|| null` redundent?

Reply:
Isn't the `|| null` redundent?

Reply:
We had a strange design where we use null instead of undefined. I was holding
off with further changes for undefined until the core PR

Reply:
All of this is moot once the final changes come in for undef usernames. The one odd thing to note is the existing intentional usage of null prior to the changes in this PR, which I assume are oversights